daemon/mgr/image.go

```diff
@@ -41,6 +41,7 @@ var acceptedImageFilterTags = map[string]bool{
 	"before":    true,
 	"since":     true,
 	"reference": true,
+	"dangling":  true,
```

Comment:
we also need to change related docs, such as swagger or cmd desc.

Reply:
Yes.